dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pPanel.vue

```diff
@@ -0,0 +1,332 @@
+<template>
+  <div v-if="waitingForData" class="d-center-div text-center px-7 py-4">
```

Comment:
We could make a component from this loading data window. No need to do here if there are more important priorities.

Reply:
Yup. Agree. Will resolve.